daily.sh

```diff
@@ -113,7 +113,11 @@ main () {
         # only try to su if we are root (or sudo)
         if [[ "$EUID" -eq 0 ]]; then
             echo "Re-running ${DAILY_SCRIPT} as ${LIBRENMS_USER} user"
-            su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            if [[ "$(uname)" == "Linux" ]]; then
+                su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            else
+                sudo -u "$LIBRENMS_USER" "$DAILY_SCRIPT"
```

Comment:
You are missing the arguments here `$@`

Also, we could probably use sudo for both Linux and FreeBSD...